### ISSUE TITLE
Accessibility: When zoomed to 400% the Proceed to Checkout button is sticky on the bottom of the cart page and blocks the content

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43637]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Low-vision users frequently zoom in on websites to be able to use them. The cart page currently has code which makes the "Proceed to Checkout" button sticky in some situations. See:
https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce-blocks/assets/js/blocks/cart/inner-blocks/proceed-to-checkout-block/block.tsx#L113

and
https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce-blocks/assets/js/blocks/cart/inner-blocks/proceed-to-checkout-block/style.scss#L24

When zooming the browser in to 400%, this blocks a significant portion of the screen (almost 30%), making it difficult for low-vision users to read the content.
<img width="1407" alt="Screenshot of sticky element" src="https://github.com/woocommerce/woocommerce/assets/13153456/f58dd8a6-214a-4051-961a-9d2568b1b10f">

### Expected behavior

When zooming the website there should be no sticky elements or elements blocking user's ability to read the content.

### Actual behavior

See above.

### Steps to reproduce

1. Open a browser window on a website using Twenty Twenty-Four and zoom your bowser to 250% or greater.
2. Add a product to the cart and visit the cart page.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.
